zoia_lib/UI/ZOIALibrarian.ui

```diff
@@ -435,7 +435,16 @@
                 <number>10</number>
                </property>
                <widget class="QWidget" name="layoutWidget">
+                <property name="sizePolicy">
```

Comment:
These sizePolicy and sizeConstraint seem to be what made the most difference in terms of stopping the layout recalc/shift